content/intro-to-storybook/angular/en/simple-component.md

```diff
@@ -273,17 +275,17 @@ As you can see, getting started building components in isolation is easy and fas
 
 ## Automated Testing
 
-Storybook gave us a great way to visually test our application during construction. The ‘stories’ will help ensure we don’t break our Task visually as we continue to develop the app. However, it is a completely manual process at this stage, and someone has to go to the effort of clicking through each test state and ensuring it renders well and without errors or warnings. Can’t we do that automatically?
+Storybook gave us a great way to manually test our application UI during construction. The `stories` will help ensure we don’t break our Task's appearance as we continue to develop the app. However, it is an entirely manual process at this stage, and someone has to go to the effort of clicking through each test state and ensuring it renders well and without errors or warnings. Can’t we do that automatically?
```

Comment:
Re: `stories` — We typically only use backticks for terms that appear verbatim in the code. I don't think that applies here, and I'm not actually sure why it was ever quoted in the first place.

Reply:
Fixed with the latest commits. Thanks for catching this!